crates/starknet-server/tests/get_transaction_receipt_by_hash.rs

```diff
@@ -18,11 +21,61 @@ mod get_transaction_receipt_by_hash_integration_tests {
     use starknet_rs_signers::{LocalWallet, SigningKey};
     use starknet_types::felt::Felt;
 
+    use crate::common::constants::CHAIN_ID;
     use crate::common::devnet::BackgroundDevnet;
     use crate::common::utils::{
         get_events_contract_in_sierra_and_compiled_class_hash, get_json_body,
     };
 
+    #[tokio::test]
+    async fn deploy_account_transaction_receipt() {
+        let devnet = BackgroundDevnet::spawn().await.expect("Could not start Devnet");
+
+        let predeployed_accounts_response =
+            devnet.get("/predeployed_accounts", None).await.unwrap();
+
+        let predeployed_accounts_json = get_json_body(predeployed_accounts_response).await;
+        let first_account = predeployed_accounts_json.as_array().unwrap().get(0).unwrap();
+
+        let private_key =
+            Felt::from_prefixed_hex_str(first_account["private_key"].as_str().unwrap()).unwrap();
+
+        // constructs starknet-rs account
+        let signer = LocalWallet::from(SigningKey::from_secret_scalar(private_key.into()));
```

Comment:
Here we are basically using the data of an already predeployed account for a newly deployed account. I was confused by this. For tests involving new account deployment, we have a utility function called `get_deployable_account_signer` which returns a dummy undeployed value.

Reply:
[3edcd76](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/149/commits/3edcd763592503fc256698434217b8cf3c4e9005)